04.ls/lib/l_option_formatter.rb

```diff
@@ -0,0 +1,69 @@
+# frozen_string_literal: true
+
+class LOptionFormatter
+  SECONDS_IN_HALF_YEAR = (365 / 2) * 24 * 60 * 60
+
+  class << self
+    def format(ls_files)
+      header = generate_header(ls_files)
+      body = generate_body(ls_files)
+      [header, *body].join("\n")
+    end
+
+    private
+
+    def generate_header(ls_files)
+      total_blocks = ls_files.map(&:blocks).sum
+      "total #{total_blocks}"
+    end
+
+    def generate_body(ls_files)
+      now = Time.now
+      max_widths = calculate_max_widths(ls_files, now)
+      ls_files.map do |ls_file|
+        formatted_file_name = format_file_name(ls_file.name, ls_file.link_name)
```

Comment:
```suggestion
        formatted_file_name = format_file_name(ls_file)
```

とした方が引数を減らせますね

Reply:
あと、ここも他と区別しなくていい名前じゃないかな？
```suggestion
        file_name = format_file_name(ls_file.name, ls_file.link_name)
```